### ISSUE TITLE
No longer able to connect Spotify account

### ISSUE THREAD

**Issue by AnalogCyan** [swdotcom/swdc-vscode-musictime#49]
**Describe the bug**
```
After authorizing Spotify, it still says "Connect Spotify". It sometimes redirects me to https://app.software.com/dashboard after authorizing with Spotify in the browser, or to https://app.software.com/login with a message telling me that this account does not exist.
```
**ISSUE TYPE**
- Bug Report

**Steps to reproduce**

1. Install Music Time
2. Try logging into Spotify
3. Get redirected/still see "Connect Spotify"

**Expected behavior**
```paste below
Connect my Spotify account, as it's successfully done in the past.
```

**ACTUAL RESULTS**

```paste below
Does not connect my Spotify account, and sometimes tells me the account doesn't exist.
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Plugin version: Code Time v2.1.4

**Comment by xavluiz** [comment]
Hi, we apologize for the connection issue. We've identified the issue and working on testing a fix for this now. We'll get let you know as soon as we release the update. Thank you for your patience.
Regards.

**Comment by xavluiz** [comment]
This should now be fixed with v2.1.8. Let us know if there's still an issue, thank you.

**Comment by AnalogCyan** [comment]
Working fine again after the update, thanks!
